folding/validators/protein.py

```diff
@@ -347,6 +357,10 @@ def save_files(
         filetypes = {}
         for filename, content in files.items():
             filetypes[filename.split(".")[-1]] = filename
+
+            if "cpt" in filename:
+                filename = "em_binary.cpt"
```

Comment:
this makes sure that we are writing to a new file. Maybe you can load this file in? but I didn't have success doing this. Maybe the combination of the above encoding will do the trick